src/OrchardCore/OrchardCore/Modules/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -126,6 +126,15 @@ private static void AddDefaultServices(OrchardCoreBuilder builder)
                 s.AddSingleton<LocalLock>();
                 s.AddSingleton<ILocalLock>(sp => sp.GetRequiredService<LocalLock>());
                 s.AddSingleton<IDistributedLock>(sp => sp.GetRequiredService<LocalLock>());
+                s.AddSingleton<CultureOptions>(sp =>
+                {
+                    var shellConfiguration = sp.GetService<IShellConfiguration>();
+                    var cultureOptions = shellConfiguration
+                        .GetSection("OrchardCore_Localization_CultureOptions")
+                        .Get<CultureOptions>();
+
+                    return cultureOptions;
+                });
```

Comment:
`builder.ConfigureServices()` is like a module startup in which we can inject services that are already registered as `ShellSettings` and `IShellConfiguration`, this even if we are building a tenant container.

In case of `builder.ConfigureServices()` you can pass to the delegate an `IServiceProvider` parameter that you can use to resolve `IShellConfiguration` and then use one of the regular `s.Configure<CultureOptions>()`.

            builder.ConfigureServices((s, serviceProvider) =>
            {
                s.AddSingleton<LocalLock>();
                s.AddSingleton<ILocalLock>(sp => sp.GetRequiredService<LocalLock>());
                s.AddSingleton<IDistributedLock>(sp => sp.GetRequiredService<LocalLock>());

                var configuration = serviceProvider.GetRequiredService<IShellConfiguration>();
                s.Configure<CultureOptions>(configuration.GetSection("OrchardCore_Localization_CultureOptions"));
            });

Note: I could have also named it `sp` in place of `serviceProvider`, that's would be okay as other `sp` are used in their own closure (if we click go to definition we can see that it makes the distinction), so I used `serviceProvider` just to make a naming distinction, but you can also use `sp`, as you want.

Reply:
I'm not sure why I didn't call `Configuure()` instead?!! May be I was sleepy ;)

Reply:
Seems I didn't notice the second overload